workshops/util.py

```diff
@@ -315,3 +318,79 @@ def merge_persons(person_from, person_to):
 
     # removes tasks, awards, qualifications in a cascading way
     person_from.delete()
+
+
+class WrongEventURL(Exception):
+    pass
+
+
+def normalize_event_index_url(url):
```

Comment:
I know that we more or like that workshop pages are on GitHub Pages but if host A want to use http://web-site-of-host-a.com/swc-workshop? Could we support this? I will not block this pull request only by this but will be nice to have it option available.